### PR TITLE
add .prebuildrc + update travis + npm script

### PR DESCRIPTION
**Pull request by ralphtheninja**
Lets be explicit in what targets we are building leveldown for. This allows a single `prebuild` command in the `leveldown` folder to build all targets.

### CODE REVIEW

**Comment by ralphtheninja** [comment]
Note, this should not affect `travis` since we don't care about any targets because `--compile` will take precedence.

**Comment by ralphtheninja** [comment]
`--verbose` only affects the initial lines, see below:

prebuild-verbose

File: package.json [hunk 1]
```diff
@@ -50,7 +50,8 @@
   },
   "scripts": {
     "install": "prebuild --download",
-    "test": "tape test/*-test.js | faucet"
+    "test": "tape test/*-test.js | faucet",
+    "prebuild": "rm -rf prebuilds && prebuild --verbose"
```
Comment (mafintosh) [inline package.json]: instead of always removing the prebuilds folder the user could just pass `--force` to force a rebuild

Comment (mafintosh) [inline package.json]: `npm run prebuild --force` - this way we still have build caching which is nice when something fails

Comment (ralphtheninja) [inline package.json]: Updating.

**Pull request merged**